docs/source/style.rst

```diff
@@ -421,6 +421,23 @@ An example of this, taken from the `Google comment conventions <https://google.g
 If a comment consists of one or more full sentences (as is typically the case for *block* comments), it should start with an upper case letter and end with a period. *Inline* comments often consist of a brief phrase which is not a full sentence, in which case they should have a lower case initial letter and not have a terminating period.
 
 
+Markup
+~~~~~~
+
+The following components require the recommended markup taken from `NumPy's Conventions <https://numpydoc.readthedocs.io/en/latest/format.html#common-rest-concepts>`_.:
+
+- Paragraphs:
+  Indentation is significant and indicates the indentation of the output. New paragraphs are marked with a blank line.
+- Variable, module, function, and class names:
+  Should be written in between single back-ticks (`x`).
+- None and NoneType:
+  Should be written in between double back-ticks (``None``).
```

Comment:
Perhaps None, NoneType, True, and False?

Reply:
Added!